src/main/java/bio/terra/common/SynapseColumn.java

```diff
@@ -117,16 +117,7 @@ static boolean checkForCollateArgRequirement(TableDataType dataType, boolean isA
   static boolean checkForJSONCastRequirement(TableDataType dataType, boolean isArrayOf) {
     if (isArrayOf) {
       return false;
-    }
-    switch (dataType) {
-      case DIRREF:
-      case FILEREF:
-      case TEXT:
-      case STRING:
-        return false;
-      default:
-        return true;
-    }
+    } else return dataType.equals(TableDataType.TEXT);
```

Comment:
Need `{}`s and newlines here. Also since `TableDataType` is an enum it's better IMO to use `==` not `equals()`
```suggestion
    } else {
      return dataType == TableDataType.TEXT;
    }
```

Reply:
Ah right.  I'll just remove the `else`